test/raygun.messageBuilder_test.js

```diff
@@ -54,7 +54,46 @@ test("basic builder tests", function (t) {
 
     var message = builder.build();
     tt.ok(message.details.error.message);
-    tt.equal("name=Test", message.details.error.message);
+    tt.equal(message.details.error.message, "name=Test");
+    tt.ok(message.details.groupingKey);
+    tt.end();
+  });
+
+  // Reference: https://github.com/MindscapeHQ/raygun4node/issues/68
+  t.test("humanise Symbol object string", function (tt) {
```

Comment:
Sorry, US English only please :-)

Humanize

Reply:
Same as in other instances below

Reply:
Will do!